docs/conf/na/2017/writing-day.rst

```diff
@@ -28,3 +28,52 @@ Documenting a new project?
 Check out our `beginners guide <http://docs.writethedocs.org/writing/beginners-guide-to-docs/>`_ to writing documentation.
 This should help you get started,
 and give you some ideas for how you can contribute to a project that you love.
+
+Write Write the Docs on Writing Day
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This year we're also running a session where you can help improve your favourite website. Yup, you can brainstorm improvements, write helpful content and posts or just magically improve `Write the Docs <http://www.writethedocs.org>`_.
+
+Ideally you'll already be familiar with GitHub and writing in plain text (markdown or restructured text), but we'll be there to help out if you're not. And if you'd rather write in Google Docs or program in python or jinja we've got you covered.
+
+Here are a few things we'll be working on on writing day, reach out to `@plaindocs <https://twitter.com/plaindocs>`_ or `@meker <https://twitter.com/meker>`_ if you have any questions or more suggestions.
+
+Brainstorm User Oriented Architechture
+=======================================
+
+Site is very content type based – find ways to introduce user-oriented labels considering audience. Who is coming to the site, why? What are the goals of the site?
+
```

Comment:
I find "content type based" vague here, but I'm having a hard time coming up with an alternative. Maybe the distinction is more between objects and actions (what's something WTD offers, vs what's something a visitor to the site might want to do)? But again, not sure how to rewrite ...